### ISSUE TITLE
generateNewView function in DoctrineCrudGenerator.php doesn't pass fieldMappings

### ISSUE THREAD

**Issue by vialcollet** [sensiolabs/SensioGeneratorBundle#325]
Hi.

I wanted to override the skeleton to include fields as form_row but discovered that the generateNewView function doesn't pass the fieldMappings to the twig template as opposed to generateEditView.

```php
protected function generateNewView($dir)
{
    $this->renderFile('crud/views/new.html.twig.twig', $dir.'/new.html.twig', array(
        'bundle'            => $this->bundle->getName(),
        'entity'            => $this->entity,
        'route_prefix'      => $this->routePrefix,
        'route_name_prefix' => $this->routeNamePrefix,
        'actions'           => $this->actions,
    ));
}
```

versus

```php
protected function generateShowView($dir)
{
    $this->renderFile('crud/views/show.html.twig.twig', $dir.'/show.html.twig', array(
        'bundle'            => $this->bundle->getName(),
        'entity'            => $this->entity,
        'fields'            => $this->metadata->fieldMappings,
        'actions'           => $this->actions,
        'route_prefix'      => $this->routePrefix,
        'route_name_prefix' => $this->routeNamePrefix,
    ));
}
```

Would it be possible to add this ? I am using Symonfy 2.3.*

For your information I had posted on [Stackoverflow ](http://stackoverflow.com/questions/27588419/how-to-loop-through-form-fields-in-sensio-generator-bundle)

Thanks for the great work. I discovered Symfony recently and it's fantastic!

**Comment by vialcollet** [comment]
No feedback?

**Comment by javiereguiluz** [comment]
Closing it as fixed by #401.

**Comment by vialcollet** [comment]
Great, thanks
